projects/demo/src/modules/components/input-date-time/index.html

```diff
@@ -115,6 +115,20 @@ <h3>DI-tokens for input-configurations:</h3>
                 is 12-hour time format with meridiem part
             </ng-template>
         </tui-doc-example>
+
+        <tui-doc-example
+            id="with-validator"
+            heading="With validator"
+            [component]="7 | tuiComponent"
+            [content]="7 | tuiExample: 'html,ts'"
+            [description]="withValidatorDescription"
+        >
+            <ng-template #withValidatorDescription>
+                Create custom validator function to verify input value. Custom
+                <code>completeDateTimeValidator</code>
+                using to check if data is complete
```

Comment:
I wonder if this description is really required 🤔

Reply:
Do you think to remove description at all?

Reply:
I looked at it again and I think it's obvious here and the description is really redundant
Removed 👍